### PR TITLE
Development

### PR DESCRIPTION
**Pull request by SUDANGOKUL**
Hey Marshall, This is the spell checker code that we all discussed, and it's completed, and working here is it.
go through IT.

### CODE REVIEW

**Review by MarshallOkafor — COMMENTED** [review]
Hi Gokul,

I have reviewed the python code you just upload. It looks good. I saw that the code can read a text file and print the result of the corrected words to another file. Awesome!

However, I noticed that the code had no comments for clarity. I would recommend that you add some comments just to make the code clearer. Once that is done, we are good to merge it to the development branch of the project.

Thanks.

File: correctedit.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from gingerit.gingerit import GingerIt
```
[the remark this answers is not in the document]
Comment (SUDANGOKUL) [reply correctedit.py:1]: sure I will do that,

File: correctedit.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+for d in corrections:
+  if d['text'] == d['correct']:
+        print()
+  else:
+        print("________________")    
+        print("Previous:",d['text'])  
+        print("Correction:",d['correct'])
+        list.append(d['correct'])        
+        
+print(list, file=open("output.txt", "a"))
```
[the remark this answers is not in the document]
Comment (SUDANGOKUL) [reply correctedit.py:21]: Thank you!

File: correctedit.py [hunk 3]
```diff
@@ -1,21 +1,21 @@
-from gingerit.gingerit import GingerIt
+from gingerit.gingerit import GingerIt #Gingerit is used to ease the file collection of dictionaries and to compare the data to the database.
```
Comment (MarshallOkafor) [inline correctedit.py:1]: Hey Gokul, I can see that we have comments now in the program.

Great! Will merger it now.

**Pull request merged**